packages/block-editor/src/components/global-styles/effects-panel.js

```diff
@@ -26,7 +26,7 @@ import { shadow as shadowIcon, Icon, check } from '@wordpress/icons';
 /**
  * Internal dependencies
  */
-import { mergeOrigins } from '../use-settings';
+import { mergeOrigins } from '../../store/private-selectors/get-inherited-settings';
```

Comment:
Maybe this should be moved to a utils file, but I left it for now, there's only two uses.